docs/content/en/docs-dev/user-guide/managing-application/defining-app-configuration/ecs.md

```diff
@@ -6,14 +6,47 @@ description: >
   Specific guide to configuring deployment for Amazon ECS application.
 ---
 
-Deploying an Amazon ECS application requires `TaskDefinition` and `Service` configuration files placing inside the application directory. Those files contain all configuration for [ECS TaskDefinition](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definitions.html) object and [ECS Service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html) object, and will be used by Piped agent while deploy your application/service to ECS cluster.
+To deploy an Amazon ECS application, `TaskDefinition` configuration file must be located in the application directory. This file contains all configuration for [ECS TaskDefinition](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definitions.html) object and will be used by Piped agent while deploy your application/service to ECS cluster.
+
+> To deploy your application to run continually or to place it behind a load balancer, You need to create [ECS Service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html). `Service` configuration file also must be located in the application directory. This file contains all configuration for [ECS Service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html) object.
+
+There are two main ways to deploy an Amazon ECS application.
+- Your application is a one-time or periodic batch job.
+  - it's a standalone task.
+  - you need to prepare `TaskDefinition`
+- Your application is deployed to run continuously or behind a load balancer.
+  - you need to prepare `TaskDefinition` and `Service`
```

Comment:
```suggestion
There are two main ways to deploy an Amazon ECS application.
- Your application is a one-time or periodic batch job.
  - it's a standalone task.
  - you need to prepare `TaskDefinition`
- Your application is deployed to run continuously or behind a load balancer.
  - you need to prepare `TaskDefinition` and `Service`

To deploy an Amazon ECS application, the `TaskDefinition` configuration file must be located in the application directory. This file contains all configuration for [ECS TaskDefinition](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/task_definitions.html) object and will be used by Piped agent while deploying your application/service to the ECS cluster.

To deploy your application to run continuously or to place it behind a load balancer, You need to create [ECS Service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html). The `Service` configuration file also must be located in the application directory. This file contains all configurations for [ECS Service](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/ecs_services.html) object.
```
I think we can revert those blocks like this to make it easy to follow

Reply:
Yes, it's easier to understand when blocks are organized🙇
I'll rewrite it🚀